frontend/src/js/controllers/authCtrl.js

```diff
@@ -55,6 +55,29 @@
             vm.wrnMsg = {};
         }
 
+        vm.passwordChecksignUp = function(password1, password2) {
```

Comment:
keep this function name 'passwordChecks' only. cause we have use this in case of reset passwords, forgot password